tests/cli/test_cli_add_relationship.py

```diff
@@ -8,7 +8,7 @@ def test_add_relationship_required(invoke_cli, disk_store: Store, helpers):
     # GIVEN a database with a sample and an family
     sample = helpers.add_sample(disk_store)
     sample_id = sample.internal_id
-    family = helpers.add_family(disk_store)
+    family = helpers.add_case(disk_store)
     family_id = family.internal_id
```

Comment:
```suggestion
    case = helpers.add_case(disk_store)
    case_id = case.internal_id
```
For the rest of the module as well

Reply:
fixed